test/brief.jl

```diff
@@ -12,7 +12,7 @@ facts("BRIEF") do
     context("Sampling Patterns") do
         # Check Bounds
         for si in [32, 128, 256], wi in [5, 15, 25, 49], se in [123, 546, 178]
-            s1, s2 = centered(si, wi, se)
+            s1, s2 = ImageFeatures.centered(si, wi, se)
```

Comment:
That would be fine, too. We could alternatively change the one in ImageFiltering, but to me (and I'm definitely biased) it seems that one is a little more general.